scripts/finalize-release.ps1

```
@@ -0,0 +1,214 @@
+    $jobs = Get-AppVeyorBuildForTag -Account $AppVeyorAccount -Project $AppVeyorProject -TagValue $Tag
+
+    $downloadDir = Join-Path $workRoot "unsigned"
+    Save-AppVeyorArtifacts -Jobs $jobs -DestinationDir $downloadDir
+
+    $targets = Get-ChildItem -Path $downloadDir -Filter "*.exe" -Recurse
+    if ($targets.Count -eq 0) {
+        throw "No exe files found after downloading artifacts."
+    }
+
+    foreach ($file in $targets) {
+        Write-Host "Signing $($file.FullName)"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'finalize-release.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)